src/runtime/lpython_builtin.py

```diff
@@ -75,3 +75,25 @@ def bool(x: i32) -> bool:
         return False
     else:
         return True
+
+
+def len(s: str) -> i32:
+    """
+    Return the length of the string `s`.
+    """
+    if s == '':
+        return 0
+    count: i32
+    count = 0
+    end: i32
+    end = -1
+    while s[count:end] != '':
+        count += 1
+    return count + 1
+
+
+def pow(x: i32, y: i32) -> i32:
+    """
+    Returns x**y.
+    """
+    return (x)**(y)
```

Comment:
I would just write it like this:
```suggestion
    return x**y
```
but it's a matter of style.